### PR TITLE
Feature more oidc modeling

### PR DESCRIPTION
**Pull request by randallmorey**
This PR completes OIDC auth method modelling with the missing string array fields.  Unlike other string array fields so far, these are updated via conventional PUT/PATCH requests, rather than via custom methods.  Thus no field-specific serialization methods are required as are found in previous serializers

### CODE REVIEW

File: addons/api/tests/unit/initializers/fragment-string-test.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+  hooks.afterEach(function () {
+    run(this.application, 'destroy');
+  });
+
+  // TODO: Replace this with your real tests.
+  test('it works', async function (assert) {
+    await this.application.boot();
+
+    assert.ok(true);
```
Comment (calcaide) [inline addons/api/tests/unit/initializers/fragment-string-test.js:30]: I guess this is the default test ember cli created?

File: addons/api/tests/unit/serializers/fragment-string-test.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+
+module('Unit | Serializer | fragment string', function (hooks) {
+  setupTest(hooks);
+
+  // Replace this with your real tests.
```
Comment (calcaide) [inline addons/api/tests/unit/serializers/fragment-string-test.js:7]: I think this comment is not valid anymore?

File: addons/api/addon/initializers/fragment-string.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+// app/initializers/fragment-serializer.js
```
Comment (calcaide) [inline addons/api/addon/initializers/fragment-string.js:1]: This comment shows where the serializer lives?

Comment (randallmorey) [reply addons/api/addon/initializers/fragment-string.js:1]: This is default generated.

File: addons/api/tests/unit/initializers/fragment-string-test.js (hunk 1 above)
Comment (randallmorey) [reply addons/api/tests/unit/initializers/fragment-string-test.js:30]: Yes.  We sometimes don't override these.

File: addons/api/tests/unit/serializers/fragment-string-test.js (hunk 2 above)
Comment (randallmorey) [reply addons/api/tests/unit/serializers/fragment-string-test.js:7]: This is generated along with the "it exists" test by default.  There are a lot of these comments hanging around.  PR to clean this up is welcome.

**Review by gsusmi — APPROVED** [review]
LGTM. Ty!

**Pull request merged**